pkg/controller/cronjob/cronjob_controllerv2_test.go

```diff
@@ -190,70 +188,573 @@ func TestControllerV2SyncCronJob(t *testing.T) {
 		jobPresentInCJActiveStatus bool
```

Comment:
you can move these variable into "cj status"?

Reply:
done